client/js/components/BeerCarousel/BeerCarousel.js

```diff
@@ -0,0 +1,202 @@
+import React, { Component } from 'react';
+import { Dimensions, Image, Text, View, TouchableOpacity } from 'react-native';
+import Carousel from 'react-native-snap-carousel';
+
+import styles from './styles';
+import { hr, center, h2, h3, Subtitle1 } from '../../config/styles';
+
+// onPress={() => {
+//   this._carousel.snapToNext();
+// }}
+
+export class BeerCarousel extends Component {
+  _renderItem({ item, index }) {
```

Comment:
if you are using _ for your methods then you should follow that convention for all your methods, throughout your project.